From 6d84a59ba7a117301e418873a29ba8fc512f1477 Mon Sep 17 00:00:00 2001
From: Pavel Valena <pvalena@redhat.com>
Date: Mon, 18 Aug 2025 16:46:38 +0200
Subject: [PATCH 26/31] test: fixup NFS test.sh

there is an additional empty line; introduced in 88796914fa725c674196c63109fdc3fdae2f0e9d
by mistake.

rhel-only

Related: RHEL-97473
---
 test/TEST-60-NFS/test.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/TEST-60-NFS/test.sh b/test/TEST-60-NFS/test.sh
index f6efea36..9d3e8306 100755
--- a/test/TEST-60-NFS/test.sh
+++ b/test/TEST-60-NFS/test.sh
@@ -377,7 +377,6 @@ test_setup() {
     "$DRACUT" -i "$TESTDIR"/overlay / \
         -a "network ${SERVER_DEBUG:+debug}" \
         -d "af_packet piix ide-gd_mod ata_piix ext4 sd_mod i6300esb virtio_net" \
-
         --no-hostonly-cmdline -N \
         -f "$TESTDIR"/initramfs.server "$KVERSION"
 }
-- 
2.52.0

